pysat/utils/io.py

```diff
@@ -382,7 +846,9 @@ def load_netcdf_pandas(fnames, strict_meta=False, file_format='NETCDF4',
                                'axis': ('axis', str), 'scale': ('scale', str),
                                'min_val': ('value_min', np.float64),
                                'max_val': ('value_max', np.float64),
-                               'fill_val': ('fill', np.float64)}):
+                               'fill_val': ('fill', np.float64)},
```

Comment:
The labels class default may need to be updated. Current default has more labels than standard pysat default. So writing and loading a file technically changes metadata.

Reply:
Sounds like this should be an issue.

Reply:
I unclearly was talking the definition right here. I can post an issue, or just, whoop whoop, take care of it here.

Reply:
Let's not make this PR any bigger 🐳

Reply:
Just big bones. 

That comment is a bit stale. Since I posted that I made an issue.